go-controller/cmd/ovnkube-trace/ovnkube-trace.go

```diff
@@ -898,6 +898,14 @@ func installOvnDetraceDependencies(coreclient *corev1client.CoreV1Client, restco
 		trueFalse := strings.TrimSuffix(depVerifyOut, "\n")
 		klog.V(10).Infof("Dependency check '%s' in pod '%s', container '%s' yielded '%s'", dependencyCmd, podName, "ovnkube-node", trueFalse)
 		if trueFalse != "true" {
+			pipDependencyCmd := "if rpm -qa | egrep -q python3-pip; then echo 'true'; fi"
```

Comment:
Use `type -p pip3` instead? `python3-pip` package name and rpm might be more platform dependent.

Reply:
oh! yes, done.